app/src/main/java/com/duckduckgo/app/systemsearch/SystemSearchViewModel.kt

```diff
@@ -202,7 +206,10 @@ class SystemSearchViewModel(
 
     fun appNotFound(app: DeviceApp) {
         command.value = Command.ShowAppNotFoundMessage(app.shortName)
-        deviceAppLookup.refreshAppList()
+
+        viewModelScope.launch(dispatchers.io()) {
```

Comment:
This was previously refreshing the app list on the UI thread

Reply:
👍